tls/s2n_resume.c

```diff
@@ -611,25 +628,50 @@ int s2n_config_wipe_expired_ticket_crypto_keys(struct s2n_config *config, int8_t
     return 0;
 }
 
+int s2n_verify_unique_ticket_key_comparator(void *a, void *b)
+{
+    return (memcmp((uint8_t *) a, (uint8_t *) b, SHA_DIGEST_LENGTH));
```

Comment:
nit: no need for additional brackets around `memcmp`

Reply:
Done.